File: packages/implementations/OneLang-Yaml-v0.1/Java/src/OneYaml.java
package io.onelang.std.yaml;

import org.yaml.snakeyaml.Yaml;
import java.util.Map;

public class OneYaml {
    public static YamlValue load(String content) {
        //console.log("loading YAML: '" + content.substring(0, 50).replace("\n", "\\n") + "'");
        Yaml yaml = new Yaml();
        Map<String, Object> obj = yaml.load(content);
        //System.out.println(obj);
        return new YamlValue(obj);
    }
}

File: packages/implementations/OneLang-Yaml-v0.1/CSharp/src/OneYaml.cs
using System.IO;
using YamlDotNet.RepresentationModel;

public class OneYaml {
    public static YamlValue load(string content)
    {
        var yaml = new YamlStream();
        yaml.Load(new StringReader(content));
        return new YamlValue((YamlMappingNode)yaml.Documents[0].RootNode);
    }
}